src/EdFi.AnalyticsMiddleTier.Common/NpgsqlConnectionStringValidator.cs

```diff
@@ -29,14 +29,6 @@ public bool IsConnectionStringValid(out string errorMessage)
                     errorMessage = $"{Environment.NewLine}Please specify a data source using the Data Source or Server keyword.";
                 }
 
-                if (!connectionStringBuilder.IntegratedSecurity &&
-                    (string.IsNullOrWhiteSpace(connectionStringBuilder.Username) ||
-                     string.IsNullOrWhiteSpace(connectionStringBuilder.Password)))
-                {
-                    errorMessage =
-                        $"{errorMessage}{Environment.NewLine}Please specify an authentication method using an integrated security or providing a user id and password.";
-                }
-
```

Comment:
`IntegratedSecurity` was removed from the library. See last line of https://www.npgsql.org/doc/security.html.